src/App.js

```diff
@@ -0,0 +1,10 @@
+import './App.css';
+import Calculator from './components/Calculator';
+
+function App() {
+  return (
+    <Calculator />
+  );
+}
+
+export default App;
```

Comment:
- [ ] Kindly make the App.js component class-based this is a requirement for this milestone 👍🏿